README.rst

```diff
@@ -60,24 +57,17 @@ Installation
 
 .. installation-start-marker
 
-Compatible with Python 3.5+:
+Compatible with Python 3.6+:
 
 .. code-block:: bash
 
     pip install dimod
 
-To install with optional components:
+To install from source (requires ``pip>=10.0.0``):
 
 .. code-block:: bash
 
-    pip install dimod[all]
-
-To install from source:
-
-.. code-block:: bash
-
-    pip install -r requirements.txt
-    python setup.py install
+    pip install .
```

Comment:
I get an error:
```
ERROR: Could not install packages due to an OSError: [('C:\\Users\\jpasvolsky\\!git_ADTT\\test_dimod\\dimod\\docs\\LICENSE', 'C:\\Users\\jpasvolsky\\AppData\\Local\\Temp\\pip-req-build-w6ovc33b\\docs\\LICENSE', 'symbolic link privilege not held'), ('C:\\Users\\jpasvolsky\\!git_ADTT\\test_dimod\\dimod\\docs\\README.rst', 'C:\\Users\\jpasvolsky\\AppData\\Local\\Temp\\pip-req-build-w6ovc33b\\docs\\README.rst', 'symbolic link privilege not held')]
```

Reply:
Incredible. Windows is why we cannot have nice things.

Reply:
For posterity, you can also do `pip install . --use-feature=in-tree-build` which will become the default behaviour in the future. For now I have reverted to the old syntax.